test/classes/range.js

```diff
@@ -82,6 +82,13 @@ test('tostrings', (t) => {
   t.end()
 })
 
+test('format', (t) => {
```

Comment:
Future devs will probably appreciate a more descriptive test here.  We know today it's for coverage of the `this.formatted` test but that's not apparent from just this test alone.

Reply:
To me this seems fine. The test is all about `.format()` calls.

Reply:
Renamed the test to "formatted value is calculated lazily and cached" and modified it to examine the `.formatted` property more directly.